Packages/com.unity.inputsystem/Extras/NPad/NPadSupport.cs

```diff
@@ -0,0 +1,71 @@
+namespace ISX.Plugins.Switch
+{
+    /// <summary>
+    /// Adds support for Switch NPad controllers.
+    /// </summary>
+    [InputPlugin]
```

Comment:
This won't compile anymore. Please remove this, move the NPad folder to Packages/com.unity.inputsystem/InputSystem/Plugins, remove the .asmdef file, and add the initialization to `InputSystem.PerformDefaultPluginInitialization`.